applications/trilinos_application/python_scripts/trilinos_strategy_python.py

```diff
@@ -146,7 +146,7 @@ def InitializeSolutionStep(self, reform_dofs):
             # reorder the list of degrees of freedom to identify fixity and system size
```

Comment:
i wonder if we should not completely remove this file (as a different pull request of course)